scenarios/basics/entity_resolution/SPECIFICATION.md

```diff
@@ -0,0 +1,289 @@
+# Specification for the AWS Entity Resolution Service Scenario
+
+## Overview
+
+This SDK Basics scenario demonstrates how to interact with AWS Entity Resolution
+using an AWS SDK. It demonstrates various tasks such as creating a schema
+mapping, creating an matching workflow, starting a workflow, and so on. Finally,
+this scenario demonstrates how to clean up resources.
+
+## Resources
+
+This Basics scenario requires an IAM role that has permissions to work with the
+AWS Entity Resolution service, an AWS Glue database and a table, and two S3
+buckets.
+A [CDK script](../../../resources/cdk/entityresolution_resources/README.md
+) is provided to create these resources.
+
+## Hello AWS Entity Resolution
+
+This program is intended for users not familiar with the AWS Entity Resolution
+Service to easily get up and running. The program uses a
+`listMatchingWorkflowsPaginator` to demonstrate how you can read through
+workflow information.
+
+## Basics Scenario Program Flow
+
+The AWS Entity Resolution Basics scenario executes the following operations.
+
+1. **Create a schema mapping**:
+    - Description: Creates a schema mapping by invoking the
+      `createSchemaMapping` method.
+    - Exception Handling: Check to see if a `ConflictException` is thrown, which
+      indicates that the schema mapping already exists. If the exception is
+      thrown, display the information and end the program.
+
+2. **Create a Matching Workflow**:
+    - Description: Creates a new matching workflow that defines how entities
+      should be resolved and matched. The method `createMatchingWorkflow` is
+      called.
+    - Exception Handling: Check to see if a `ConflictException` is thrown, which
+      is thrown if the matching workflow already exists. ALso check to see if a `ValidationException` is thrown. If so, display the message and end the program.
+
+3. **Start Matching Workflow**:
+    - Description: Initiates a matching workflow by calling the
+      `startMatchingJob` method to process entity resolution based on predefined
+      configurations.
+    - Exception Handling: Check to see if an `ConflictException` is thrown,
+      which indicates that the matching workflow job is already running. If the
+      exception is thrown, display the message and end the program.
+
+4. **Get Workflow Job Details**:
+    - Description: Retrieves details about a specific matching workflow job by
+      calling the `getMatchingJob` method.
+    - Exception Handling: Check to see if an `ResourceNotFoundException` is
+      thrown, which indicates that the workflow cannot be found. If the
+      exception is thrown, display the message and end the program.
+
+5. **List Matching Workflows**:
```

Comment:
The code that runs says "5. Get the schema mapping for the JSON data.", which doesn't match this step in the Spec.

Reply:
The Spec now specifies: 

5. **Get Schema Mapping**:

Which reflects the code in the scenario.